include/pfs/drp/stella/Controls.h

```diff
@@ -255,9 +259,9 @@ struct FiberTraceProfileFittingControl {
         profileInterpolation("SPLINE3"),
         swathWidth(500),
         telluric("NONE"),
+        overSample(10),
```

Comment:
This is a member ordering change.  Does it now agree with the declaration order (see MeyersEffective C++  if you're confused)